server/src/test/java/com/netcracker/MessageTest.java

```diff
@@ -0,0 +1,31 @@
+package com.netcracker;
+
+import com.netcracker.domain.Message;
+import org.junit.Test;
+
+import static com.netcracker.domain.MessageType.NAME_REQUEST;
+import static com.netcracker.domain.MessageType.TEXT;
+import static org.junit.Assert.assertEquals;
+
+public class MessageTest {
+    @Test
+    public void shouldReturnMessageTypeWhenTryingToGet() {
```

Comment:
Also just a suggestion to make your names more finished:
shouldReturnMessageTypeWhenTryingToGet (when trying to get what ?) -> shouldReturnMessageTypeWhenAskingForIt